src/bot/transactionTableRow.ts

```diff
@@ -41,7 +40,10 @@ export function tableRow(
     memo: tx.memo ?? "",
     category: tx.category ?? "",
     account: tx.account,
-    hash: TRANSACTION_HASH_TYPE === "moneyman" ? tx.uniqueId : tx.hash,
+    hash:
```

Comment:
Don't forget to update the pullrequest's description to match the fix you actually did